package/network/utils/comgt/files/directip.sh

```diff
@@ -45,7 +45,7 @@ proto_directip_setup() {
 	}
 
 	cardinfo=$(gcom -d "$device" -s /etc/gcom/getcardinfo.gcom)
-	[ -n $(echo "$cardinfo" | grep -q "Sierra Wireless") ] || {
+	$(echo "$cardinfo" | grep -q "Sierra Wireless") || {
```

Comment:
Shouldn't we change the whole line to something like

`gcom -d "$device" -s /etc/gcom/getcardinfo.gcom | grep -q "Sierra Wireless" || {`

I mean the cardinfo variable is never set to local as it should be and not really used